cmake/modules/RootNewMacros.cmake

```diff
@@ -1356,10 +1356,15 @@ endfunction()
 # ROOT_ADD_PYUNITTESTS( <name> )
 #----------------------------------------------------------------------------
 function(ROOT_ADD_PYUNITTESTS name)
+if(MSVC)
```

Comment:
This we also need for the current PyROOT, right?

Reply:
The current PyROOT doesn't work on Windows